### DISCUSSION TITLE
Adding a new model to an existing migration scripts

### DISCUSSION THREAD

**Discussion by Princedave101** [miguelgrinberg/Flask-Migrate#543] [General]
Goodday, 
      i have a question, i had a migration already initialized for my project and added a model called User to represent my table  and after i migrated and upgraded it. it created the database file and everything properly on my system. so i discovered i needed a post model(table) for my blog app and i decided to migrate and upgrade i didnt have any error until i tried checking my post table from flask shell and it said it wasn't created , does flask migrate create other tables after initial migration or i have to create it manually

**Comment by miguelgrinberg** [comment]
Have you created a second migration? Each time you change your models you have to generate a new migration with `flask db migrate`, and then apply it with `flask db upgrade`.
